[2022-06-14 16:43:26,706] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-06-14T16:43:22.432043+00:00 [queued]>
[2022-06-14 16:43:26,724] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-06-14T16:43:22.432043+00:00 [queued]>
[2022-06-14 16:43:26,726] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-14 16:43:26,726] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-14 16:43:26,727] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-14 16:43:26,746] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): run_after_loop> on 2022-06-14 16:43:22.432043+00:00
[2022-06-14 16:43:26,752] {standard_task_runner.py:52} INFO - Started process 2943 to run task
[2022-06-14 16:43:26,756] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2022-06-14T16:43:22.432043+00:00', '--job-id', '14', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpwer6ee89', '--error-file', '/tmp/tmph1v70let']
[2022-06-14 16:43:26,759] {standard_task_runner.py:80} INFO - Job 14: Subtask run_after_loop
[2022-06-14 16:43:26,839] {task_command.py:370} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2022-06-14T16:43:22.432043+00:00 [running]> on host a7436a86ff1e
[2022-06-14 16:43:27,164] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2022-06-14T16:43:22.432043+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-14T16:43:22.432043+00:00
[2022-06-14 16:43:27,168] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-14 16:43:27,170] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2022-06-14 16:43:27,193] {subprocess.py:85} INFO - Output:
[2022-06-14 16:43:27,196] {subprocess.py:92} INFO - 1
[2022-06-14 16:43:27,198] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-14 16:43:27,248] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20220614T164322, start_date=20220614T164326, end_date=20220614T164327
[2022-06-14 16:43:27,292] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-14 16:43:27,367] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
